### ISSUE TITLE
Clearing localStorage on each test run

### ISSUE THREAD

**Issue by whw** [keathley/wallaby#86]
Is there a way to clear local storage for each test run? Or to specify different local storage directories? 

I see that it's possible to modify phantomjs to use a different local storage path, but I'm trying to avoid tests that modify local storage from stepping on one another.

**Comment by keathley** [comment]
If you just need to clear local storage after each test then you could try something like this:

```elixir
defmodule MyTests do
  setup %{session: session} do
    on_exit fn ->
      execute_script(session, "localStorage.clear()")
    end

    :ok
  end
end
```

There currently isn't a way to specify the local storage directory. That would be a relatively easy thing to add in the future. On the other hand it might be worthwhile to clear out local storage whenever wallaby creates a new session with phantom. There *shouldn't* be an issue doing this since we isolate phantoms anyway.

**Comment by whw** [comment]
Thanks!

Clearing local storage can create problems with multiple testing running in parallel.  Better to just give each isolated phantom it's own local storage directory I think: https://github.com/keathley/wallaby/pull/93

**Comment by keathley** [comment]
Is the issue with using JS to clear localstorage that they're all using a shared directory or something? I'm not opposed to using isolated storage directories. Just wondering what the root problem is.

Sorry it took me so long to respond to this. ElixirConf had all my attention for a while :)

**Comment by whw** [comment]
No problem - I hope ElixirConf was fun and engaging!

Yeh, basically each process is using the same, default localstorage directory. So clearing the directory (via JS) can be bad if another process was expecting the directory to be non-empty.

**Comment by keathley** [comment]
This should be fixed with #93
